src/read.c

```diff
@@ -4050,7 +4050,7 @@ static avifResult avifParse(avifDecoder * decoder)
             AVIF_CHECKRES(
                 avifParseMovieBox(data, boxOffset, boxContents.data, boxContents.size, decoder->imageSizeLimit, decoder->imageDimensionLimit));
             moovSeen = AVIF_TRUE;
-            decoder->imageSequenceTrackPresent = AVIF_TRUE;
+            decoder->imageSequenceTrackPresent = (decoder->data->tracks.count > 0);
```

Comment:
Done. We don't need the `hasTrak` variable. we can simply check tracks.count since avifParseTrackBox always creates a track on success.

Also added a test.